ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -100,7 +100,7 @@
    * The maximum number of parallel requests allowed for a delete operation.
    */
   @Config("router.delete.request.parallelism")
-  @Default("12")
+  @Default("3")
```

Comment:
I didn't see a test in `DeleteManagerTest` that tests the case where the local colo says NOT_FOUND and the calls go remote and succeed (and don't succeed). If we are making this change, that should be added.

There is also no `DeleteOperationTest` :( Thats orthogonal to this patch though.